src/window.rs

```diff
@@ -386,6 +388,21 @@ impl WindowBuilder {
         self
     }
 
+    /// Build window with parent window.
+    ///
+    /// The default is `None`.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Windows** : A child window has the WS_CHILD style and is confined to the client area of its parent window.
+    /// For more information, see <https://docs.microsoft.com/en-us/windows/win32/winmsg/window-features#child-windows>
+    /// - **Android / iOS / Wayland:** Ignored.
+    #[inline]
+    pub fn with_parent_window(mut self, parent_window: Option<RawWindowHandle>) -> Self {
```

Comment:
Since we're taking a `RawWindowHandle` directly here, shouldn't this technically be marked `unsafe`?

Reply:
Yes, this is true. I was thinking about having `&impl AsRawWindowHandle` as argument, but the issue is that user may want to send window to other thread and then create a new one from the main thread with the parent that you've already sent or your window might be not present.

I'd suggest to mark it `unsafe` then, and say that raw-window-handle must point to the valid window. Enforcing life time is not required here I guess?